cotainr/tests/cli/test_info.py

```diff
@@ -127,6 +130,17 @@ def test_found_singularity(self, monkeypatch):
             == "Found singularity 3.7.4-1 >= 3.7.4, \x1b[92mOK\x1b[0m"
         )
 
+    def test_found_singularity_ce(self, monkeypatch):
```

Comment:
I am not sure whether these tests are well-designed. We could reduce them to one with parameterization.

Reply:
I will add a task to the backlog to refactor these tests. I think they are fine for now.